cypress/integration-qe/e2e-enterClusterSpecs.spec.js

```diff
@@ -58,7 +68,7 @@ describe('Enter cluster details', () => {
   it('can select the first subnet CIDR', () => {
     cy.get('#form-input-hostSubnet-field')
       .find('option')
-      .then(($els) => $els.get(0).setAttribute('selected', 'selected'))
+      .then(($els) => $els.get(1).setAttribute('selected', 'selected'))
```

Comment:
Such approach works but it is low-level with assumptions about implementation.

It is not a must, but with cypress we can mimic user's clicking on elements.
I guess it will be result in more code but closer to reality.